recipes/sentry-crashpad/all/test_package/CMakeLists.txt

```diff
@@ -9,3 +9,22 @@ find_package(crashpad REQUIRED CONFIG)
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE crashpad::client)
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+
+if(APPLE)
```

Comment:
These should be set in the recipe ... components have https://github.com/conan-io/conan-center-index/blob/be8e7c051e59bfa712a18160a76b23927285813b/recipes/gainput/all/conanfile.py#L79

Reply:
Thanks, fixed.